src/passwordless/serialization.py

```diff
@@ -221,8 +222,24 @@ class DeleteUserSchema(Schema):
     user_id = fields.Str(data_key="userId", required=True)
 
 
-class SendMagicLinkRequestSchema(Schema):
+class SendMagicLinkOptionsSchema(Schema):
     email_address = fields.Str(data_key="emailAddress", required=True)
     url_template = fields.Str(data_key="urlTemplate", required=True)
     user_id = fields.Str(data_key="userId", required=True)
     time_to_live = fields.Int(allow_none=True)
+
+
+class GenerateAuthenticationTokenOptionsSchema(Schema):
+    user_id = fields.Str(data_key="userId", required=True)
+    time_to_live = fields.Int(allow_none=True)
```

Comment:
is it `time_to_live` or `timeToLive` ? 
Same question for `SendMagicLinkOptionsSchema`